### PR TITLE
PWN-3974 - fix token History by reloading if no transactions

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-3974

## Description of Work
Fix token History by reloading if no transactions

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt [hunk 1]
```diff
@@ -94,7 +94,10 @@ class TokenHistoryPresenter(
             if (isRefresh) {
                 transactions.clear()
             }
-            val fetchedItems = historyInteractor.loadTransactions(token.publicKey, isRefresh)
+            val fetchedItems = historyInteractor.loadTransactions(
+                token.publicKey,
+                isRefresh = isRefresh || transactions.isEmpty()
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt:99]: I am not sure if it's a best solution. 
cannot get how this field effects the empty state

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt (hunk 1 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt:99]: yes i agree
i'm consulting with @davran1001 
but it looks like issue in caches of states
and reloading fixes it

i will spend more time to check the root cause of this issue

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt:99]: then it's wip?

Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/history/ui/token/TokenHistoryPresenter.kt]: fixed @1sakov

**Review by davran1001 — APPROVED** [review]

**Review by 1sakov — APPROVED** [review]

**Pull request merged**